tensorflow/lite/kernels/internal/reference/reduce.h

```diff
@@ -354,11 +354,11 @@ inline bool QuantizedMeanOrSum(const T* input_data, int32 input_zero_point,
   }
 
   // Calculate mean by dividing output_data by num of aggregated element.
-  U num_elements_in_axis = 1;
+  size_t num_elements_in_axis = 1;
```

Comment:
is there a particular error that the template param was causing? If not, we should keep the template param.

Reply:
Got a compilation error for comparing signed and unsigned. Removed this change and fixed it in function call

Reply:
Still got compilation errors for comparing signed int and unsigned int with google internal build toolchain.
Asked to the original authors about this "U num_elements_in_axis = 1;" line and they agreed size_t is better for this variable.
Could you add this change again?

Reply:
Done. Added this change again in latest commit.